modules/EnergyManager/EnergyManager.hpp

```diff
@@ -16,12 +16,11 @@
 // headers for required interface implementations
 #include <generated/energy/Interface.hpp>
 
+// ev@4bf81b14-a215-475c-a1d3-0a484ae48918:v1
```

Comment:
why did this file got changed?

Reply:
Somebody accidentially removed the guarding tags earlier